apps/teams/backends.py

```diff
@@ -183,7 +183,7 @@ def get_permissions(self):
         [
             AppPermSetDef("experiments", ALL),
             AppPermSetDef("bot_channels", ALL),
-            AppPermSetDef("annotations", ALL),
+            AppPermSetDef("annotations", [VIEW]),
```

Comment:
To get the write permissions for creating tags you'll need to add a `ModelPermSetDef`:

```python
ModelPermSetDef("annotations", "customtaggeditem", ALL)
ModelPermSetDef("annotations", "usercomment", ALL)
```